### DISCUSSION TITLE
How to count the results?

### DISCUSSION THREAD

**Discussion by smile2cmd** [facebookresearch/detectron2#2497] [Q&A]
Hy everybody, we want to count the result labels on the picture with the example colabBook and get an integer for every class which is detected. For example, 2 persons, 1 horse, and so on. How can we do that?

**Comment by ppwwyyxx** [comment]
https://detectron2.readthedocs.io/modules/structures.html#detectron2.structures.Instances should have enough information to parse the outputs of models
